mytk/tests/testImages.py

```diff
@@ -0,0 +1,62 @@
+import envtest
+import unittest
+import os
+from mytk import *
+import tempfile
+import collections
+import random
+import pathlib
+
+class TestImage(unittest.TestCase):
+    def setUp(self):
+        self.app = App(geometry="500x300")
+        self.delegate_function_called = False
+        self.resource_directory = pathlib.Path(__file__).parent.parent / "resources"
+
+    def tearDown(self):
+        self.app.quit()
+
+    def start_timed_mainloop(self, function=None, timeout=500):
+        if function is not None:
+            self.app.root.after(int(timeout/4), function)
+        self.app.root.after(timeout, self.app.quit) # max 5 seconds
```

Comment:
this function seems to be needed in every unit test might be cool at some point to centralize it in a kind of testUtils file or something like that

Reply:
Yes: I will derive a class from unites.testCase and include that function and others.  I was reaching that point, but has now become obvious it is needed.